EveripediaNetwork/ep-ui
# Footer and Create Wiki button fixes

# Fix Careers Link and Create Wiki Button Issues

_Ensures the Careers link opens in a new page and fixes the 'Create Wiki' button so that the entire button, not just the text, is clickable._

## How should this be tested?

1. Careers Link:

> - On the footer, click on the Careers link.
> - Verify that the link opens in a new tab/page.

2. Create Wiki Button:

> - On the Navbar, click on the 'Create Wiki' button.
> - Verify that the entire button is clickable, not just the text.

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/3059, closes https://github.com/EveripediaNetwork/issues/issues/3060
